packages/react-composites/tests/browser/call/hermetic/CallControls.test.ts

```diff
@@ -31,3 +32,29 @@ test.describe('Call composite custom button injection tests', () => {
     expect(await stableScreenshot(page)).toMatchSnapshot(`custom-buttons.png`);
   });
 });
+
+test.describe('Call composite custom call control options tests', () => {
+  test('Control bar buttons correctly show as compact with camera disabled and end call button hidden', async ({
+    page,
+    serverUrl
+  }) => {
+    const initialState = defaultMockCallAdapterState([defaultMockRemoteParticipant('Paul Bridges')]);
+    const testOptions: CallCompositeOptions = {
+      callControls: {
+        displayType: 'compact',
+        cameraButton: {
+          disabled: true
+        },
+        microphoneButton: true,
```

Comment:
What does `microphoneButton: true` signify as the microphone is disabled as well in the snapshots

Reply:
They are simply valid options so including them in the test, we actually have a bug where setting it to true was causing it to become disabled: #2325